components/formats-bsd/src/loci/formats/in/OMETiffReader.java

```diff
@@ -1056,7 +1063,14 @@ public MetadataStore getMetadataStoreForConversion() {
     int realSeries = getSeries();
     for (int i=0; i<getSeriesCount(); i++) {
       setSeries(i);
-      store.setPixelsBinDataBigEndian(new Boolean(!isLittleEndian()), i, 0);
+      if (meta.getPixelsBinDataCount(i) > 0) {
+        for (int j=0; j<meta.getPixelsBinDataCount(i); j++) {
+          store.setPixelsBinDataBigEndian(new Boolean(!isLittleEndian()), i, j);
+        }
+      }
+      else {
+        store.setPixelsBigEndian(Boolean.TRUE, i);
```

Comment:
I think that if such an image did exist, it would not be read successfully by OmeTiffReader.  It doesn't support pixel data except if in a plane referred to by TiffData, and we would have mangled the BigEndian attribute based upon the endianness of the TIFF container when we set it on Pixels and then on each BinData element.  So I think it's probably safe to assume that BinData isn't handled properly at all by this reader.

I'd be more interested to know specifically which code is dependent upon *consuming* this BinData.BigEndian attribute, assuming that there is a consumer and this isn't merely historical cruft.